examples/subsampling/monitor.cxx

```diff
@@ -31,7 +28,7 @@ class Monitor1dDump : public Monitor {
         output[item.name].attributes["time_dimension"] = "t";
       }
     }
-    output_file.write(output);
+//    output_file.write(output);
```

Comment:
Again, the data should be dumped, otherwise it is rather useless.

Or did you test that?